src/ER_V/acl/acl.h

```diff
@@ -0,0 +1,50 @@
+/**
+ * ACL Conversion Module
+ * 
+ * This module is responsible for the conversion of received movement commands to ACL
+ * ACL is the programming language we use to communicate with the Scorbot-ER V
+ */
+
+#pragma once
+
+#include <stdint.h>
+
+#include "api/api.h"
+#include "data/s_list.h"
+
+#define ACL_SIZE 32
+
+
+/** ACL Command Map for the command string, length, and node */
+typedef struct _acl_command
+{
+  char        payload[ACL_SIZE]; /** The ACL command (dynamically sized)*/
+  uint8_t     len;               /** The length of the ACL command */
+  S_List_Node node;              /** The node of the s_list */
+} ACL_Command;
+
+/**
+ * @brief Allocates memory for an ACL_Resources' array of ACL_Commands
+ * @details Iterates through command pool in global ACL_Resources and populates with dummy data
+ * @returns 1 on success
```

Comment:
The standard for the Operator system is return -1 on failure, and typically 0 on success (if the return is only used for determining success; otherwise return whatever it's supposed to)